controller/users.go

```diff
@@ -600,6 +625,104 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 	return returnResponse
 }
 
+func (c *UsersController) createAuthUpdateClient(ctx *app.UpdateUsersContext) (*authservice.Client, error) {
+	authEndpoint, err := c.config.GetAuthEndpointUsers(ctx.Request)
+	if err != nil {
+		return nil, err
+	}
+
+	u, err := url.Parse(authEndpoint)
+	if err != nil {
+		return nil, err
+	}
+	client := authservice.New(goaclient.HTTPClientDoer(http.DefaultClient))
+	client.Host = u.Host
+	client.Scheme = u.Scheme
+	client.SetJWTSigner(goasupport.NewForwardSigner(ctx))
+	return client, nil
+}
+
+func (c *UsersController) updateInAuth(ctx *app.UpdateUsersContext) (*app.User, error) {
```

Comment:
This too should be consolidated and maintained as part of a separate interface spec , something like `RemoteAuthService` - but definitely fine for now.